AcadeMeData/models.py

```diff
@@ -199,3 +199,52 @@ def __str__(self):
 
     def get_msg_tag(id):
         return MessageTags.objects.get(id=id)
+
+
+class Course(models.Model):
+    course_id = models.IntegerField(primary_key=True, validators=[MinValueValidator(0)], default=0)
+    name = models.CharField(max_length=100)  # Name of the course
+    degree = models.ForeignKey(Degree, on_delete=models.RESTRICT)
```

Comment:
does each course belong to a single degree?

Reply:
How can we store multiple foreign keys?
We were having the same issue with the 'Degree' model, where we couldn't link multiple universities to a degree..

Reply:
maybe one to many relations? https://docs.djangoproject.com/en/3.2/ref/models/fields/

Are you trying to read django docs?

Reply:
There is no One-to-many relation in the link you've posted..
From a quick google search it says that One-to-many field is defined as a ForeignKey, where one table is connected to many different tables.
This is of course not the situation we have here, as we want a set of foreign keys which I don't understand how to implement.

Reply:
IMO this case is many-to-many, as every degree has multiple courses, and every course can appear in multiple degrees.

usually we create a db table that handles it, like "course in degree", but with django `[ManyToManyField](https://docs.djangoproject.com/en/3.2/ref/models/fields/#manytomany-arguments)` this table is created and managed automatically.

Reply:
Nice! It took a while but finaly I understood how to implement a ManyToMany field.
Now every course has a set of degrees that offer this specific course. :)